lib/share_progress/facebook_variant.rb

```diff
@@ -0,0 +1,23 @@
+require 'share_progress'
+require 'share_progress/variant'
+
+module ShareProgress
+  class FacebookVariant < Variant
+    attr_accessor :facebook_title, :facebook_thumbnail, :facebook_description
+
+    def initialize(facebook_title:, facebook_description:, facebook_thumbnail:, button:)
+      @facebook_title = facebook_title
```

Comment:
Are you sure they're required by SP? It's not clear to me from [the spec](https://docs.google.com/document/d/1f1Jktt4gk9b3qugH8h-3bj6DoijQ3_6-RxUw5uq7DWU/edit#heading=h.66d9v1vnsmth) that they are. If they are, I agree we should enforce them, but for something like `facebook_thumbnail` I can see leaving that blank.